src/components/cesium_map/api/spatial.js

```diff
@@ -303,18 +303,32 @@ export class ISamplesSpatial {
     });
 
     this.viewer = new Cesium.Viewer(element, {
-      infoBox:false,
+      // infoBox:false,
       timeline: false,
       animation: false,
       sceneModePicker: false,
       terrainProvider: worldTerrain,
+      fullscreenElement : element
     });
     this.buildingTileset = this.viewer.scene.primitives.add(Cesium.createOsmBuildings());
     this.handler = new Cesium.ScreenSpaceEventHandler(this.viewer.canvas);
     this.viewer.scene.globe.depthTestAgainstTerrain = true;
     this.mouseCoordinateCallback = null;
     this.selectBoxCallback = null;
     this.selectedBox = null;
+
+    // entity label for point primitive identifier
+    this.pointLabel = this.viewer.entities.add({
+      label: {
+        show: false,
+        showBackground: true,
+        font: "14px monospace",
+        horizontalOrigin: Cesium.HorizontalOrigin.LEFT,
+        verticalOrigin: Cesium.VerticalOrigin.BOTTOM,
+        pixelOffset: new Cesium.Cartesian2(15, 0),
+        disableDepthTestDistance: 1.2742018 * 10 ** 7
```

Comment:
This magic number should be extracted to a constant with a symbolic name explaining what it represents.